packages/block-editor/src/components/rich-text/README.md

```diff
@@ -121,14 +121,22 @@ registerBlockType( /* ... */, {
 } );
 ```
 
+## RichText.isEmpty
+
+Helper function to test if supplied variable (e.g. content of RichText) is empty. It tests if value of variable is empty or if variable is an empty array.
+
+### `value: String | Array`
```

Comment:
Array is deprecated so I wouldn't document it here. Note that it can also be `RichTextData`

Reply:
Sorry, I had urgent matters and I forgot about this request - it seems to me that it could be `RichTextValue`, not `RichTextData`, but only in React native version. Deprecation notice on array is also only specified in React native version. Nevertheless, I agree with your comments.